src/lib/Zikula/Bundle/CoreBundle/DependencyInjection/Compiler/DoctrinePass.php

```diff
@@ -1,4 +1,15 @@
 <?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
```

Comment:
I don't care about this really. eventually we need to replace the header entirely.